### ISSUE TITLE
Roma and bin directory of Stylesheets

### ISSUE THREAD

**Issue by MarkAsmar** [TEIC/Stylesheets#287]
I am using the Roma CLI version in order to generate schemas, however, when I use teitorelaxng in the bin directory of the stylesheets I get another output. It is a bigger file with like a 1000 lines larger than the one generated using the Roma. However I used the HTML documentation to check if the schema generated by Roma is correct and in fact it is. Then I used both files to validate MEI files (I am generating schemas out of the MEI schema), they both validated all files the same way. 
What is the difference? I don't know if that is a bug! Does it cause a certain problem using one of the two approaches?

**Comment by martindholmes** [comment]
Did you try diffing the files? As far as I understand it, the cli Roma is no longer supported and hasn't been updated in a long while; the other process is presumably grabbing the latest version of TEI, which may have a lot of new stuff in it, and hence the resulting schema might be larger. If the files you're validating don't include any of the newer features, then they would validate with both schemas. But this is a guess...

**Comment by MarkAsmar** [comment]
I've been told by @jamescummings that Roma is still supported. And actually, when using roma2.sh in [https://github.com/TEIC/Roma](url) you can specify what stylesheet you want to use. I used the latest stylesheets, and generated schemas in both ways. I diffed the files and they are different. However, when validating MEI files, those MEI files do contain the elements I added myself in the ODD file and in both ways they validated correctly exactly the same.

**Comment by ahankinson** [comment]
I would appreciate some direction from the TEI community on this as well, on behalf of the MEI community. I'm under the impression that Roma has been deprecated in favour of the TEI Stylesheets tools, and so we've been advising individuals to use the TEI Stylesheets project for their customizations.

**Comment by lb42** [comment]
Roma uses the same tei stylesheet library as everything else. The question is which version of that library you're picking up. It is usually displayed as a comment at the start of
Roma outputs.

**Comment by martindholmes** [comment]
If you use the XML Diff function in Oxygen, what are the significant differences between the schemas?

**Comment by jamescummings** [comment]
@MarkAsmar @ahankinson: I don't believe that roma2.sh is maintained any more. Roma (the php web app) is maintained but not developed (i.e. critical bugs will be fixed but we aren't adding new features). The OxGarage service is maintained and supported (but we probably don't want to add lots of new features...) It is what Roma uses to do its transformations. But that is just another interface for the underlying Stylesheets which are maintained and developed.  The stylesheets and the bin/teitoX Ant tasks are what we are primarily recommending for programmatic transformations (whether TEI ODD to RelaxNG or DOCX to TEI, etc.)  Even the new RomaJS will be using the underlying Stylesheets here, all of the various bits of software do (Roma, roma2.sh, bin/teitorelaxng (etc).  They are the real product in my mind.  The thing with Roma (web app) is that you can't do lots of things _in it_ that you can do in the TEI ODD language.  You can always edit your file and then upload it there and transform it to a schema (the Stylesheets should be the same, so the result should be the same if using the last release of the Stylesheets).
